### ISSUE TITLE
Loading Movenet gives Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'loadGraphModel') but it did not before

### ISSUE THREAD

**Issue by MarioCavero** [tensorflow/tfjs#7930]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow.js):
- OS Platform and Distribution (Linux Ubuntu 22.04):
- TensorFlow.js installed from (npm or script link): installed via npm

**Describe the current behavior**
I am working with the pose estimation model from movenet, lightinging single pose. It was working until this morning. It is giving an error when loading the model, which did not happen before. The line of error according to the console is in pose-detection:17:
```
pose-detection:17 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'loadGraphModel')
    at pose-detection:17:56068
    at pose-detection:17:2398
    at Object.next (pose-detection:17:2503)
    at pose-detection:17:1440
    at new Promise (<anonymous>)
    at u (pose-detection:17:1185)
    at Xt (pose-detection:17:55433)
    at Object.<anonymous> (pose-detection:17:71747)
    at pose-detection:17:2398
    at Object.next (pose-detection:17:2503)
```
Going to the source:
```
case 2:
                    return a = void 0,
                    "SinglePose.Lightning" === e.modelType ? a = "https://tfhub.dev/google/tfjs-model/movenet/singlepose/lightning/4" : "SinglePose.Thunder" === e.modelType ? a = "https://tfhub.dev/google/tfjs-model/movenet/singlepose/thunder/4" : "MultiPose.Lightning" === e.modelType && (a = "https://tfhub.dev/google/tfjs-model/movenet/multipose/lightning/1"),
                    [4, i.loadGraphModel(a, {
                        fromTFHub: o
 })];
```

**Describe the expected behavior**
It should load the model without any problems, as these past weeks. 

**Standalone code to reproduce the issue**
Model in my .js file
```
await tf.setBackend('webgl');
    await tf.ready();

    const detectorConfig = {
      modelType: poseDetection.movenet.modelType.SINGLEPOSE_LIGHTNING,
    };
    const detector = await poseDetection.createDetector(poseDetection.SupportedModels.MoveNet, detectorConfig);

```
and the importing in the html file:
```
  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"></script>
  <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/pose-detection"></script>
```
  This is the model link, it seems it is working.
  
`https://tfhub.dev/google/tfjs-model/movenet/singlepose/lightning/4`

I will try to load the model locally transforming the model via tfjs converter to get the json file and binaries but why was this working before and is not working now?

**Comment by MarioCavero** [comment]
Thanks for the quick response @gaikwadrahul8 . The only problem regarding the way you loaded it (it did not throw any errors to me btw! Thanks!) is that I do not know the method to do inferences, and if it follows the same architecture as the 1st one I was using. 
I will post it as you say in the TF hub repo, and wait for them to reply! I will post an answer (if any) in here! 
Thanks again for such a quick response!

**Comment by cuspace** [comment]
Got the same error on the js code loading on the browser with html and script tag

**Comment by emsoneal** [comment]
I'm also getting the same error. Would love any information on how to fix it!

**Comment by no-made** [comment]
Got the same error in the same behaviuor of @cupspace

**Comment by aviban12** [comment]
Got the same error with webgl, Would love any information about the fix

**Comment by gaikwadrahul8** [comment]
Hi, @cuspace, @emsoneal, @no-made, @aviban12 

Could you please help me with steps and reproducible code to reproduce the same issue from my end ? Thank you!

**Comment by t1n3** [comment]
thank you @gaikwadrahul8 for looking after the problem! 

i got the same issue. at the moment i'm working with a workaround in embedding an older version of the tensorflow pose-detection-model locally. maybe helpful for someone else too while waiting for the solution, attached as txt-file: 

[tensorflow-models-pose-detection-20230824.txt](https://github.com/tensorflow/tfjs/files/12468206/tensorflow-models-pose-detection-20230824.txt)

**Comment by gaikwadrahul8** [comment]
Hi, @MarioCavero, @cuspace, @emsoneal, @no-made, @aviban12

I have noticed this issue is happening with latest versions of `@tensorflow-models/pose-detection@2.1.1 `and `@tensorflow-models/pose-detection@2.1.2` and it's working as expected with this version `@tensorflow-models/pose-detection@2.1.0` so as temporary workaround could you please use `@tensorflow-models/pose-detection@2.1.0` version, team is working to fix this issue with latest versions, once I got the update from concerned team, I'll update the same here

If issue still persists with `@tensorflow-models/pose-detection@2.1.0` version, let us know with error log to investigate this issue further. Thank you


script Tag :
`<script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/pose-detection@2.1.0"></script> `

Thank you for support and cooperation

**Comment by fengwuyao** [comment]
Hi! The issue should be fixed with npm version@2.1.3. Feel free to try out with 
script tag:
`<script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/pose-detection@2.1.3"></script> ` And let us know if the issue still persist.

**Comment by sidsingla** [comment]
Hi, I am getting the same error with my custom models running until yesterday!
Could anyone help please?
In Safari: Error: An SSL error has occurred and a secure connection to the server cannot be made....
In Chrome: Error: Failed to fetch...

Thanks!
